### PR TITLE
Added a way to get attachments

### PR DESCRIPTION
**Pull request by Nyholm**
Related to #398 

I do not want to create a generic "get" function. That is why I added some regex. However, im not sure the regex is okey. This needs some testing.

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. Of course I should add some tests. This needs to be rebased on #478

File: tests/Api/AttachmentTest.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
```
Comment (DavidGarciaCat) [inline tests/Api/AttachmentTest.php]: Hey @Nyholm 

```
0.65s$ $TEST_COMMAND
> vendor/bin/phpunit --testsuite unit && vendor/bin/phpunit --testsuite functional
PHP Warning:  Unsupported declare 'strict_types' in /home/travis/build/mailgun/mailgun-php/tests/Api/AttachmentTest.php on line 3
PHP Stack trace:
```

Thoughts?

Comment (Nyholm) [reply tests/Api/AttachmentTest.php]: Oh. That should be removed.

**Comment by DavidGarciaCat** [comment]
Hey @Nyholm 

PR makes sense to me but due Travis tests have failed I'm unable to merge...

**Comment by Nyholm** [comment]
Yeah. Thank you. 
I guess I have to update the regex. Will do it later.

**Comment by Nyholm** [comment]
Wohoo. Travis is happy

**Comment by DavidGarciaCat** [comment]
Yep! sorry for the delay on that merge!

**Pull request merged**